testsuite/MDAnalysisTests/analysis/test_dssp.py

```diff
@@ -13,15 +13,52 @@
     "pdb_filename", glob.glob(f"{DSSP_FOLDER}/?????.pdb.gz")
 )
 def test_file_guess_hydrogens(pdb_filename, client_DSSP):
+    # run 2.9.0 tests (which include PRO)
+    # ignore_proline_donor=False
+    # TODO: update reference data for ignore_proline_donor=True
```

Comment:
We should really have correct reference data. About half of the files do not show a difference between `ignore_proline_donor=True` and `ignore_proline_donor=False`.